ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__periodic.yaml

```diff
@@ -36,6 +51,29 @@ tests:
     name: osde2e-credentials
   - mount_path: /usr/local/osde2e-rosa-stage
     name: osde2e-rosa-stage
+- as: ocp414-compat
+  cron: 0 8 * * 4
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - as: obo-e2e
+      cli: latest
+      commands: |
+        if ! which kubectl; then
+            mkdir -p /tmp/obo
+            export PATH=$PATH:/tmp/obo
+            ln -s "$(which oc)" /tmp/obo/kubectl
```

Comment:
how about just setting an alias?

Reply:
Test alias by run the following, it failed to work.
```
[root@617b9606e971 origin]# alias
alias kubectl='oc'
alias mv='mv -i'
alias rm='rm -i'
[root@617b9606e971 origin]# kubectl version
Client Version: 4.14.0-ec.4
Kustomize Version: v5.0.1
[root@617b9606e971 origin]# cat run-e2e-test.sh 
#!/usr/bin/bash
set -e -u -o pipefail
kubectl
[root@617b9606e971 origin]# ./run-e2e-test.sh 
./run-e2e-test.sh: line 3: kubectl: command not found
```

Reply:
you're probably missing `shopt -s expand_aliases` in the script so it would use the aliases defined, but as is just getting things more complex let's revert this to your original soft link. Thanks!

Reply:
thanks